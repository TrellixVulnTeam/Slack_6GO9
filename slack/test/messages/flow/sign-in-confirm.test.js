const SignInConfirm = require('../../../lib/messages/flow/sign-in-confirm');

describe('SignInConfirm rendering', () => {
  test('works', async () => {
    const message = new SignInConfirm('U12345', 'https://github.com/tarebyte', 'tarebyte');
    expect(message.getAttachment()).toMatchSnapshot();
  });
});
