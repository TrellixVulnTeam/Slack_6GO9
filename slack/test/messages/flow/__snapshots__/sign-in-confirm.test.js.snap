// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignInConfirm rendering works 1`] = `
Object {
  "color": "#24292f",
  "mrkdwn_in": Array [
    "text",
  ],
  "text": ":white_check_mark: Success! <@U12345> is now connected to <https://github.com/tarebyte|@tarebyte>",
}
`;
